packages/gallery/tests/components/gallery/slideshowView.spec.js

```diff
@@ -204,6 +208,54 @@ describe('Slideshow View', () => {
     });
   });
 
+  describe('Virtual Item Loading', () => {
+    it('only load items that are visible', () => {
+      Object.assign(initialGalleryViewProps.options, {
+        behaviourParams: {
+          gallery: {
+            horizontal: {
+              itemVirtualization: {
+                enabled: true,
+                rightItemMargin: 0,
+                leftItemMargin: 0,
+              },
+            },
+          },
+        },
```

Comment:
Please don't duplicate tests and change something small. I can't understand the difference between the tests without reading it slowly. The difference between tests should be understood from the code, not from the `it`